sotodlib/tod_ops/azss.py

```diff
@@ -6,6 +6,7 @@
 from sotodlib import core, tod_ops
 from sotodlib.tod_ops import bin_signal, apodize, filters
 import logging
+import so3g
```

Comment:
where is so3g being used exactly? None of the changes in this file add the use of so3g

Reply:
Taken away